Eduria/Eduria/Controllers/LecturersController.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Eduria.Models;
+using Eduria.Services;
+using EduriaData.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Eduria.Controllers
+{
+    public class LecturersController : Controller
+    {
+        private UserService Service { get; set; }
+
+        public LecturersController(UserService service)
+        {
+            Service = service;
+        }
+        /// <summary>
+        /// returns the index page with an overview of all lecturers
+        /// </summary>
+        /// <returns></returns>
+        public IActionResult Index()
+        {
+            IEnumerable lecturers = Service.GetAllUsersByUserType((int)UserRoles.Docent);
+            List<UserModel> lecturerList = new List<UserModel>();
+            foreach(User lecturer in lecturers)
+            {
+                UserModel model = new UserModel()
```

Comment:
Ik zou hier persoonlijk een converter methode van maken waar je in dit geval een User object meestuurd en een UserModel object teruggeeft. Die kun je dan direct in lijn 37 plaatsen als volgt: lecturerList.Add(ConvertToUserModel(User));